$OpenBSD: patch-src_base_h,v 1.1 2012/03/31 11:10:57 naddy Exp $

Do not redefine "deprecated".  It is not used anywhere in IceWM and
breaks the expansion of _X_DEPRECATED in <X11/Xlib.h>.

--- src/base.h.orig	Sat Mar 31 03:07:20 2012
+++ src/base.h	Sat Mar 31 03:07:33 2012
@@ -1,12 +1,6 @@
 #ifndef __BASE_H
 #define __BASE_H
 
-#if ( __GNUC__ == 3 && __GNUC_MINOR__ > 0 ) || __GNUC__ > 3
-#define deprecated __attribute__((deprecated))
-#else
-#define deprecated
-#endif
-
 /*** Atomar Data Types ********************************************************/
 
 #ifdef NEED_BOOL
